src/libvirtApi/common.js

```diff
@@ -21,57 +21,104 @@
  * Provider for Libvirt using libvirt-dbus API.
  * See https://github.com/libvirt/libvirt-dbus
  */
-import cockpit from 'cockpit';
+import cockpit from "cockpit";
+import store from "../store.js";
+import * as python from "python.js";
+import * as service from "service.js";
 
-import store from './store.js';
+import getOSListScript from "raw-loader!../getOSList.py";
 
 import {
     undefineNetwork,
     undefineStoragePool,
     undefineVm,
     updateLibvirtVersion,
-    updateOrAddStoragePool,
+    updateOsInfoList,
     updateVm,
+    setLoggedInUser,
     setNodeMaxMemory,
-} from './actions/store-actions.js';
+} from "../actions/store-actions.js";
 
 import {
     getLibvirtServiceState,
     getRefreshInterval,
     usagePollingEnabled
-} from './selectors.js';
-import VMS_CONFIG from "./config.js";
+} from "../selectors.js";
+import VMS_CONFIG from "../config.js";
 import {
     logDebug,
-} from './helpers.js';
+} from "../helpers.js";
 
-import {
-    unknownConnectionName,
-} from './libvirt-common.js';
 import {
     domainGet,
     domainGetAll,
-} from './libvirtApi/domain.js';
+    getPythonPath,
+} from "../libvirtApi/domain.js";
 import {
     call,
     dbusClient,
     Enum,
     timeout,
-} from './libvirtApi/helpers.js';
+} from "../libvirtApi/helpers.js";
 import {
     interfaceGetAll,
-} from './libvirtApi/interface.js';
+} from "../libvirtApi/interface.js";
 import {
     networkGet,
     networkGetAll,
-} from './libvirtApi/network.js';
+} from "../libvirtApi/network.js";
 import {
     nodeDeviceGetAll,
-} from './libvirtApi/nodeDevice.js';
+} from "../libvirtApi/nodeDevice.js";
 import {
     storagePoolGet,
     storagePoolGetAll,
-} from './libvirtApi/storagePool.js';
+} from "../libvirtApi/storagePool.js";
+
+/**
+ * Calculates disk statistics.
+ * @param  {info} Object returned by GetStats method call.
+ * @return {Dictionary Object}
+ */
+function calculateDiskStats(info) {
+    const disksStats = {};
+
+    if (!("block.count" in info))
```

Comment:
Please avoid such unrelated changes as " -> ' in huge commits like this - because it makes the diff really hard to follow. 
For your interest I review with:  git diff 61fb78edd67e769d8de300572add15c7c64367de..HEAD --color-moved --ignore-space-change

Which is pretty easy to review - but such small unrelated changes breal the flow.